<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>action</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>allDeliveries</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>author</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>book</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>logs</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>marketBook</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>staffMember</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>typeStyle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>bookView</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Изменить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Печать</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>country</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>Alex</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>France</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>VLadislav</string>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>Russia</string>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>Donald</string>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string>US</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuDatabase"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
